examples/heating_and_cooling/src/run_case.py

```diff
@@ -0,0 +1,148 @@
+import logging
+from pathlib import Path
+from unittest import TestCase
+
+from mesido.esdl.esdl_parser import ESDLFileParser
+from mesido.esdl.profile_parser import ProfileReaderFromFile
+from mesido.util import run_esdl_mesido_optimization
+from mesido.workflows.utils.adapt_profiles import (
+    adapt_hourly_year_profile_to_day_averaged_with_hourly_peak_day,
+)
+
+logger = logging.getLogger("WarmingUP-MPC")
+logger.setLevel(logging.INFO)
+
+
+class HeatColdDemand(TestCase):
+
+    def heating_cooling_case(self):
+        """
+        In this case we have a network with an air-water hp, a WKO (warm and cold well) and both
+        hot and cold demand. The heat and cold demand was balanced such that the seasonal storage
+        gets utilized as intended.
+        """
+        import os
+        import sys
+
+        root_folder = os.path.join(Path(__file__).resolve().parent.parent.parent.parent, "tests")
+        sys.path.insert(1, root_folder)
+
+        from models.wko.src.example import HeatProblem
+        from utils_tests import demand_matching_test, energy_conservation_test
+
+        base_folder = Path(__file__).resolve().parent.parent
+
+        # ------------------------------------------------------------------------------------------
+        class HeatingCoolingProblem(HeatProblem):
```

Comment:
A general check on this test/example case. Why are we not inheriting from the EndScenarioSizing problem class and then adding the additional cyclic constraint on the WKO (which we later have to add to EndScenarioSizing anyways? as well as overwriting the selection of the peak day from EndScenarioSizing in read?

Reply:
There was not specific reason for this that I can remember. If we want to change this I suggest we should create an issue for this so that this PR can be completed.

Reply:
Created an issue